### PR TITLE
Fixed IPFS lock issue

### LINKED ISSUE #205: Fix fast restarts of AD4M launcher causes error

### PR DESCRIPTION
**Pull request by fayeed**

### CODE REVIEW

**Review by leifriksheim — DISMISSED** [review]

File: ui/src-tauri/src/main.rs [hunk 1]
```diff
@@ -70,9 +70,10 @@ fn main() {
         let _ = remove_dir_all(data_path());
     }
 
-    while data_path().join("ad4m").join("ipfs").join("repo.lock").exists() {
-        std::thread::sleep(std::time::Duration::from_secs(1));
-        println!("IPFS repo.lock exists, waiting...");
+    while data_path().join("ipfs").join("repo.lock").exists() {
```
Comment (lucksus) [inline ui/src-tauri/src/main.rs:74]: Looks like removing "ad4m" in this line might be the fix already? I'm a bit weary of always and immediately deleting the lock...

Comment (lucksus) [reply ui/src-tauri/src/main.rs:74]: Pushed a change.

**Review by lucksus — APPROVED** [review]

File: ui/src-tauri/src/main.rs (hunk 1 above)
Comment (fayeed) [reply ui/src-tauri/src/main.rs:74]: Actually tried just doing that but it didn't fix it for me, could be just because of windows.

Comment (fayeed) [reply ui/src-tauri/src/main.rs:74]: Also, I checked the IPFS go implementation there was an issue there too for this and they just deleted the IPFS lock.

Comment (lucksus) [reply ui/src-tauri/src/main.rs:74]: So do you think we should revert this? Could you try on your windows machine? This is working for me on macOS.

Comment (fayeed) [reply ui/src-tauri/src/main.rs:74]: This works it's just that for me on windows the lock never releases, it could just be a windows thing.

**Pull request merged**